app/src/main/java/com/github/sdp/ratemyepfl/auth/ConnectedUser.kt

```diff
@@ -4,9 +4,22 @@ package com.github.sdp.ratemyepfl.auth
 Interface providing information about the current user
  */
 interface ConnectedUser {
+    /**
+     * Return the user status
+     *
+     * @return true if the user is logged in, false otherwise
```

Comment:
Yess :) I had it because the FakeConnectedUser was not following the implementation I thought it was (e.g., return null elements when the user is not logged in). Thus, to avoid any confusion, I add it once and for all.